### PR TITLE
feat: frontend part of clear general cache

### PR DESCRIPTION
**Pull request by lukicenturi**
Frontend part of #5998

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/composables/api/session/index.ts [hunk 1]
```diff
@@ -22,8 +23,23 @@ export const useSessionApi = () => {
     return handleResponse(response);
   };
 
+  const refreshGeneralCacheTask = async (): Promise<PendingTask> => {
+    const response = await api.instance.post<ActionResult<PendingTask>>(
+      '/cache/general/refresh',
+      {
+        async_query: true
+      },
```
Comment (kelsos) [inline frontend/app/src/composables/api/session/index.ts:31]: 👀 just for consistency can you do it they way we do it in other places?

**Comment by kelsos** [comment]
@lukicenturi I can see how purge and this are similar, but as a user, I would never think of searching in the purge menu to find this.

Can you add a new card/area for Cache Refresh, which we can evolve in the future?

It will be a bit better from a discoverability perspective.

**Comment by lukicenturi** [comment]
> @lukicenturi I can see how purge and this are similar, but as a user, I would never think of searching in the purge menu to find this.
> 
> Can you add a new card/area for Cache Refresh, which we can evolve in the future?
> 
> It will be a bit better from a discoverability perspective.

Updated!

**Comment by kelsos** [comment]
So two things first it's good to abstract the code, but let's not mix the naming.

second for refresh the trash icon doesn't make that much sense so it would be better if we could replace it with a refresh icon there.

File: frontend/app/src/components/settings/data-security/data-management/RefreshCache.vue [hunk 2]
```diff
@@ -0,0 +1,89 @@
+<script setup lang="ts">
+import { type Ref } from 'vue';
+import { OtherPurge } from '@/types/session/purge';
+
+const { tc } = useI18n();
+const purgable = [
+  {
+    id: OtherPurge.GENERAL_CACHE,
```
[the remark this answers is not in the document]
Comment (lukicenturi) [reply frontend/app/src/components/settings/data-security/data-management/RefreshCache.vue:8]: Yeah it makes sense now in the morning 😛

**Comment by lukicenturi** [comment]
> So two things first it's good to abstract the code, but let's not mix the naming.
> 
> second for refresh the trash icon doesn't make that much sense so it would be better if we could replace it with a refresh icon there.

Updated now

**Review by kelsos — APPROVED** [review]
lgtm @lukicenturi thanks

**Pull request merged**